k8sfv/leastsquares_test.go

```diff
@@ -17,6 +17,7 @@ package main
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
```

Comment:
I do `make go-fmt`, which just runs `goimports` in the build container.  Maybe a version mismatch?